File: mullins-roger/century-from-year.js
// Use Math.floor(year/10)
//
// BEGIN
// SET century = year/10 rounded to int
// SET comparison range for century
// IF year falls in range
// RETURN century + 1
// ELSE
// RETURN century
// END

function century(year) {
  // Finish this :)
  let result = 0;
  let century = Math.floor(year/100);
  let testCase = century*100;
  console.log('Year: ' + year);
  console.log('Century: ' + century);
  console.log('Test Case: ' +testCase);
  console.log('Range: ' + (testCase+1) + "-" + (testCase+100));

  if ((year >= (testCase+1)) && (year <= (testCase+100))) {
      result = century + 1;
      } else {
      result = century;
      };
  console.log(result);
  
  
  return result;
}

File: mullins-roger/century-from-year.py
def century(year):
    # Finish this :)
    print("Year in: ", year)
    test = year // 100
    print(year, " mod 100 = ", test)
    test2 = year % 100
    print(test2, " should be in range 0 - 100")
    if test2 >= 1 and test2 <= 100:
        print("Returning ", test+1)
        return test+1
    else:
        print("Returning ", test)
        return test
